### ISSUE TITLE
We're sending `treatment_complete` messages twice

### ISSUE THREAD

**Issue by ritazagoni** [OpenFn/Miracle-Feet#61]
## Describe the bug
See issue #32 for the latest specs.

If a treatment is completed, the `treatment_complete` alert is pushed twice to `alertsToSend`. I think [here](https://github.com/OpenFn/Miracle-Feet/blob/1acab3d8e15a2e6cb583f02c0ae62ca6dd627208/schedule_sms.js#L259) and also [here](https://github.com/OpenFn/Miracle-Feet/blob/1acab3d8e15a2e6cb583f02c0ae62ca6dd627208/schedule_sms.js#L295-L296).

## To Reproduce
1. Here is a link to a failed run on OpenFn.org wihch is indicative of the bug: https://www.openfn.org/projects/pdbznd/runs/r6g4kam8

### expression.js
Link to the job itself in Github: https://github.com/OpenFn/Miracle-Feet/blob/master/schedule_sms.js

### state.json
See `LP` for Infobip portal

## Expected behavior
The `treatment_complete` message is only scheduled once.

## To test/resolve
1. After the desired output is working locally (from the CLI), please push commits to master.
2. Please test the change on OpenFn.org by re-running this run https://www.openfn.org/projects/pdbznd/runs/r6g4kam8 and confirming success.

**Comment by lakhassane** [comment]
You're right @ritazagoni and thanks for flagging the exact line to look at :D.
I made the corrections
